homeassistant/components/atome/sensor.py

```diff
@@ -137,13 +144,19 @@ def day_price(self):
         """Return latest daily usage value."""
         return self._day_price
 
+    @property
+    def day_lastreset(self):
```

Comment:
Please separate words with underscore in variable names etc.
```suggestion
    def day_last_reset(self):
```

Reply:
Thats how I wrote it in the begining but was stuck with a bug... I will revert to this syntax as the bug I was facing was not syntax related!